gcloud/streaming/transfer.py

```diff
@@ -812,7 +1155,7 @@ def _send_chunk(self, start):
             end = body_stream.stream_end_position
             if body_stream.stream_exhausted:
                 self.__total_size = end
-            # TODO: Here, change body_stream from a stream to a string object,
+            # Here, change body_stream from a stream to a string object,
```

Comment:
Was this `TODO` addressed? If no, shouldn't the comment be removed and a bug filed?